packages/utils/README.md

```diff
@@ -0,0 +1,64 @@
+# Form5 / Utils
+
+## Demo
+
+[https://form5.github.io/](https://form5.github.io/)
+
+## Quick start
+
+```console
+$ npm i @form5/utils
+```
+
+OR
+
+```console
+$ npx jsr add @form5/utils
```

Comment:
in first part you souldn't try to support jsr. they not supporting JSX out of the box (for moment). and building then publishing have some issues (Lucas is on it)